### PR TITLE
add default language and score

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

File: src/pytest/test_followhashes.py [hunk 1]
```diff
@@ -17,7 +19,7 @@ def testSyntax1(env):
             
     env.expect('ft.create', 'idx2',
             'LANGUAGE', 'eng' 
-            'SCHEMA', 'foo', 'text').equal('Unknown argument `foo`')
+            'SCHEMA', 'foo', 'text').equal('Invalid language')
```
Comment (MeirShpilraien) [inline src/pytest/test_followhashes.py:22]: Why was this changed?

File: src/pytest/test_followhashes.py [hunk 2]
```diff
@@ -214,3 +216,32 @@ def testSortable(env):
 def testMissingArgs(env):
     env.expect('FT.CREATE', 'idx', 'ON', 'SCHEMA', 'txt', 'TEXT', 'num', 'NUMERIC').error()
     env.expect('FT.CREATE', 'idx', 'ON', 'HASH', 'FILTER', 'SCHEMA', 'txt', 'TEXT', 'num', 'NUMERIC').error()
+
+def testWrongArgs(env):
+    env.expect('FT.CREATE', 'idx', 'SCORE', 'SCHEMA', 'txt', 'TEXT', 'num', 'NUMERIC').error()
```
Comment (MeirShpilraien) [inline src/pytest/test_followhashes.py:221]: Can you verify the error message with .contains(<part of the error message>)

File: src/pytest/test_followhashes.py (hunk 1 above)
Comment (ashtul) [reply src/pytest/test_followhashes.py:22]: before language was the name of the field so any input was valid.

**Comment by ashtul** [comment]
> Small comments,
> I also do not see where you save the default score and default lang to rdb.

Good point.

**Comment by MeirShpilraien** [comment]
> Good point.

I also think its a good idea to add a test that makes sure the default score and lang are saved after rdb load.

**Comment by rafie** [comment]
@ashtul Can we squeeze in the documentation? Thanks!
